compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -35,11 +35,11 @@
 namespace luci
 {
 
-bool is_valid(const circle::OperatorCodeT &opcode);
-bool is_custom(const circle::OperatorCodeT &opcode);
-std::string opcode_name(const circle::OperatorCodeT &opcode);
-const char *tensor_name(const circle::TensorT &tensor);
-const circle::QuantizationParametersT *tensor_quantization(const circle::TensorT &tensor);
+bool is_valid(const circle::OperatorCode *opcode);
```

Comment:
could you please explain why this is needed?

Reply:
If you ask about removing functions from old API, I mark this changes as optional. I guess reviewer will check introduced changes by commits to better understanding, but if this changes makes confuse, I could remove them from draft.

If you ask about functions duplication, I made copies of functions which works with pointers from direct access API instead UnPacked since I would like to minimize usage of UnPacked API with this draft without changes in logic of importer. 

-------------

If I haven't answered your question, could you please clarify it? =)

Reply:
Sorry for ambigious question..
You've change from using reference to pointer. I guess this is not to use `Unpack`..